radio/src/targets/common/arm/stm32/adc_driver.cpp

```diff
@@ -88,32 +89,35 @@ void adcInit()
 
   ADC_MAIN->CR1 = ADC_CR1_SCAN;
   ADC_MAIN->CR2 = ADC_CR2_ADON | ADC_CR2_DMA | ADC_CR2_DDS;
-  ADC_MAIN->SQR1 = (NUM_ANALOGS_ADC-1) << 20; // bits 23:20 = number of conversions
+  ADC_MAIN->SQR1 = (NUM_ANALOGS_ADC-1) << 20; // bits 23:20 = number of conversions, NUM_ANALOGS_ADC
 
 #if defined(PCBX10)
   if (ANALOGS_PWM_ENABLED()) {
-    ADC_MAIN->SQR2 = (ADC_CHANNEL_EXT1<<0) + (ADC_CHANNEL_EXT2<<5); // conversions 7 and more
+    ADC_MAIN->SQR2 = (ADC_CHANNEL_EXT1<<0) + (ADC_CHANNEL_EXT2<<5) + (ADC_Channel_TempSensor<<10) + (ADC_Channel_Vrefint<<15) + (ADC_Channel_Vbat<<20);      ; // conversions 7 and more
```

Comment:
Only ADC1, not ADC3

Reply:
Yes when I did that X12 and X10 was a bit fuzzy and I did not have HW to test on, so that might be wrong.

Reply:
No problem. Also on this board you cannot measure the RTC and the temperature in the same time as they share the same input. I am a little bit lazy, I will do the minimum